spec/db_connection_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'db_connection'
```

Comment:
A very elegant way of testing this class. I like your use of expect....to receive.